src/DynamoCore/Scheduler/UpdateRenderPackageAsyncTask.cs

```diff
@@ -240,23 +241,56 @@ private void GetTessellationDataFromGraphicItem(Guid outputPortId, IGraphicItem
                         if (factory.TessellationParameters.ShowEdges)
                         {
                             //TODO if we start to instance more types, expand this edge generation.
+                            //and the swtich case below.
                             if (graphicItem is Topology topology)
                             {
-                                var edges = topology.Edges;
```

Comment:
It looks like previously `edges` was non-null for any topology whereas now it is non-null for only cuboids?

Reply:
cuboids are the only solid which can be instanced currently.

Reply:
What about spheres? I thought you have spheres in the screenshot?

Reply:
Scratch that, spheres don't have edges in our case.